### ISSUE TITLE
add :accessibilityStatement either to BIBO or DCTerms

### ISSUE THREAD

**Issue by kcoyle** [dcmi/dc-srap#79]
No usable property was found.

This is needed to inform potential users of accessibility features.  
**Recommendation:**

* Add accessibilityStatement either to BIBO or DCTerms.

**Comment by kcoyle** [comment]
I'm not sure that we made the distinction between "accessible = you have the rights" vs "accessible = can be used with a screen reader." There was a discussion in [February](https://github.com/dcmi/dc-srap/blob/main/meetings/2025-02-25%20_srap_meeting_57.md) that seems to have concluded that we should leave this ambiguous. I'll see if I can find more.

**Comment by niklasl** [comment]
Possibly relevant: https://schema.org/accessibilityFeature (and https://schema.org/accessibilityHazard).

Note also that in the Swedish union library catalogue (Libris), we are planning to use (and/or define) IRIs for the terms in the [a11y vocabulary](https://www.w3.org/community/reports/a11y-discov-vocab/CG-FINAL-vocabulary-20241209/), for improved usage and application support (by having labels and meanings of each term in different languages; and possible broader implications, including coordination with media/carrier types). This is an argument for allowing both text values and IRIs for this property.

**Comment by osma** [comment]
This was definitely intended to represent accessibility in the WCAG sense, for example adequate contrast and screen reader support.

If the `Statement` suffix has problematic connotations, I think we could rename it to something different. For example `accessibilitySummary` would align with MARC 532 subfield $a, which is called "Summary of accessibility" and defined as

> Text describing accessibility features, hazards, and deficiencies as well as technical details relating to accessibility features.

**Comment by annakasprzik** [comment]
accessibilityNote?

**Comment by osma** [comment]
Note: We didn't discuss in today's review meeting whether this property should go to BIBO or DCTerms. Personally, I think this would be a good fit for DCTerms because it's very general. Anyway, we need to make a decision on this as well (unless changing DCTerms is strictly off-limits...)

**Comment by kallimathios** [comment]
Language and framing from the IFLA Accessibility Metadata & Accessible Formats in Working Group may be helpful: https://docs.google.com/document/d/1IrGR0vQeKIfEhltJj4TSvy4-1DNSncH6pfH87sXrEY8/edit?tab=t.0

Also referenced: 
- https://www.loc.gov/marc/bibliographic/bd532.html
- https://commission.europa.eu/strategy-and-policy/policies/justice-and-fundamental-rights/disability/union-equality-strategy-rights-persons-disabilities-2021-2030/european-accessibility-act_en
- https://www.levelaccess.com/compliance-overview/european-accessibility-act-eaa/

**Suggested definition**: information about the accessibility of a resource

**Comment by annakasprzik** [comment]
So "capabilities" is not just an attribute of humans? That's what I associate first.

**Comment by kallimathios** [comment]
I like this @annakasprzik but in some cases the note is not only about enhancements; it also contains hazards or technical details. It enhances communication more than anything else. This is why I tried to be rather vague but I was concerned my attempt wasn't detailed enough

**Comment by annakasprzik** [comment]
accessibilityNote, **suggested definition**: "information on features of the resource concerning its accessibility for users"

(features can also be flaws?)

**Comment by kallimathios** [comment]
accessibilityNote: A human-readable summary of technical features that assist or impede universal access to a resource.

I am not able to formulate an opinion on whether or not it belongs in DC or BIBO yet :)

**Comment by kcoyle** [comment]
> A statement of technical features of the resource that assist or impede universal access.

1. May be a URI if the data is located elsewhere, so may not be strictly human-readable
2. Might have to be more than a summary - could include detailed instructions
3. I like the concepts around "assist or impede" and am just not sure of the wording
4. I like the use of "universal access"

**Comment by kcoyle** [comment]
There is stable, and then there is stagnant. To me, stable means that we will not change existing terms in a way that would invalidate current metadata using that term. The example that we are living with at the moment is the range of `dct:identifier` (literal) - changing the range would cause problems for many uses of DC Terms. Adding new terms to DC Terms will not invalidate existing metadata profiles because the new terms will not be present and can be ignored if not needed. Communities can begin using these terms in their own time, including never. If we do add terms (which will not be often) we can announce that on the DC site and to known mailing lists, first "for comment" and then as approved.  

I agree that we must be cautious, but that the vocabulary should not be frozen.

**Comment by nishad** [comment]
I am sorry that my statement wasn’t clear. I was not arguing against adding new terms to DCTerms. In fact, I believe that adding any non-conflicting terms will not break the stability.

What I was pointing out is that DCTerms is not just a vocabulary; it is a formal standard, itself standardized, including but not limited to ISO 15836-2. (ISO 15836-1 and NISO Z39.85, etc., are only about the core terms.)

Adding anything to DCTerms should be done after a thorough assessment, unlike BIBO. This affects a lot of large organisations that have stable frameworks and processes based on DCTerms-related standards.

**Comment by tombaker** [comment]
To be clear, I agree that `accessibilityNote` belongs in DC Terms - ideally.

We have discussed various definitions on Zoom calls only to find that real accessibility experts disagree with our wordings. My concern was that creating a property useful beyond the use case of bibliographic resources - and getting it right, with input from the accessibility community or with a more thorough assessment - could distract from the task of approving SRAP as a whole.

If we resolve, as a group, to go for a more generally applicable property in DC Terms, and we agree that we need expert feedback before finalizing a decision, then it becomes a question of process. Perhaps that process could be as simple as announcing the proposal on the DC site and to known mailing lists, first for comment and then as approved - as @kcoyle suggests. I will discuss a way forward with Argie and Niklas.

**Comment by tombaker** [comment]
@philbarker You suggested [in December](https://github.com/dcmi/dc-srap/issues/79#issuecomment-3636586514):
> Is there any appetite for using the [dcx:](http://purl.org/dcx/) for terms that are considered extra or extensions or externally defined but relevant to DCTerms? Currently LRMI is the only vocabulary in this namespace (edit: i.e. http://purl.org/dcx/lrmi-terms/ ).

As I [just posted](https://github.com/dcmi/dc-srap/issues/93#issuecomment-3823703881) to another issue, the URI `http://purl.org/dcx/` has never been directly used as a namespace URI. Rather, it was used to form namespace URIs for LRMI and LRMI concept schemes:
- http://purl.org/dcx/lrmi_terms/
- http://purl.org/dcx/lrmi-vocabs/interactivityType/
- http://purl.org/dcx/lrmi-vocabs/alignmentType/
- http://purl.org/dcx/lrmi-vocabs/educationalAudienceRole/
- http://purl.org/dcx/lrmi-vocabs/educationalUse/

It would IMO create confusion if DCMI were to start adding properties and classes directly to the `http://purl.org/dcx/` as a new namespace URI. There is enough confusion with `/dc/terms/` and `/dc/elements/1.1/`.

**Comment by osma** [comment]
The Finnish recommendation for metadata in institutional repositories nowadays contains two metadata elements:

- [accessibilityfeature](https://www.kiwi.fi/spaces/Julkaisuarkistopalvelut/pages/409534977/dc.description.accessibilityfeature) - a more structured representation with a small vocabulary of recommended values (all of which correspond to one of the values for the [Schema.org accessibility vocabularies](https://www.w3.org/2021/a11y-discov-vocab/latest/) - but using just one property instead of several as in Schema.org)
- [accessibilitysummary](https://www.kiwi.fi/spaces/Julkaisuarkistopalvelut/pages/409534980/dc.description.accessibilitysummary)  - a free text field for representing aspects that go beyond the structured values

Possible values for `accessibilityfeature`:

## Heading Levels Tagged

Used when the heading levels of the material accurately reflect the content hierarchy and enable smooth navigation with assistive technologies.
Corresponds to the Schema.org value structuralNavigation.

## Alternative Textual Descriptions for Images

Used when alternative descriptions have been provided for image content.
Corresponds to the Schema.org value alternativeText.

## Logical Reading Order

Used when the reading order of the content is clearly marked.
Corresponds to the Schema.org value readingOrder.

## MathML

Used when mathematical or chemical formulas have been produced using the MathML markup language.
Corresponds to the Schema.org value MathML.

## Mathematical Formulas Described

Used when mathematical or chemical formulas have been provided with textual equivalents.
Corresponds to the Schema.org value describedMath.

## Captions for Hearing Impaired

Used when synchronized captions are available for the publication's audio or video content and cannot be turned off. If a user can turn captions on and off, this value is not given. Instead, a note is made in the dc.description.accessibilitysummary field.
Corresponds to the Schema.org value openCaptions.

## Tagged PDF

Used when the structures of the PDF file's content (paragraphs, lists, tables, etc.) are marked with tags that assistive technologies can utilize.
Corresponds to the Schema.org value taggedPDF.

## Unknown Accessibility

Used when the accessibility features of the material are not known. This value should not be used in conjunction with any other value.
Corresponds to the Schema.org value unknown.

## Significant Lack of Accessibility

Used when there are significant accessibility shortcomings in the publication. This value should not be used in conjunction with any other value.
Corresponds to the Schema.org value none.

**Comment by kcoyle** [comment]
This is a good time to remind folks that DCTAP has the concept of "picklists" for values, and that a controlled vocabulary can be a set of literals, like @osma 's example above. That works for relatively small sets of values, but as I understand it @philbarker has developed DCTAPs with large lists. It's awkward in a table format, but we shouldn't assume that "control" necessarily means IRI-identified terms.

**Comment by niklasl** [comment]
For comparison, using JSON-LD, it is possible to define the scope of values for a term (using a local context with it's own set of terms and/or default vocabulary space), so that what _appears_ to be a a set of strings is mapped to IRI:s.

For example, with a context like:
```json
{
  "@context": {
    "@vocab": "http://example.org/",
    "accessibilityFeature": {"@type": "@vocab", "@context": {"@vocab": "http://example.net/a11y/"}}
  }
}
```
this:
```json
{
  "@id": "http://example.com/resource",
  "@type": "EBook",
  "accessibilityFeature": ["largePrint", "highContrastDisplay"]
}
```
means, exactly:
```turtle
prefix : <http://example.org/>
prefix a11y: <http://example.net/a11y/>

<http://example.com/resource> a :EBook ;
  :accessibilityFeature a11y:largePrint , a11y:highContrastDisplay .
```

That means that you can both "shrink to fit" the data and still have controlled terms in the linked data sense; for keeping with the follow-your-nose principle. That is valuable for having one uniform way of looking up definitions of terms without having "out of band" rules for each case. It may be simple enough, but may also lead to reliance on more advanced machinery to do the disambiguation and verification. That of course depends on the context in which the data lives. (And for some things (e.g. dates), datatyped values is reasonably the more pragmatic way (since the value space is huge/infinite).)

A related, recurring question is where to specify "error correction", i.e. how is the contract defined for what is a "clear enough" value. In the example here, "kuvilla vaihtoehtoiset kuvaukset" and "alternativa textuella beskrivningar för bilder" mean the exact same thing ("alternative textual descriptions for images"), _perhaps_ more exactly [something](https://www.w3.org/community/reports/a11y-discov-vocab/CG-FINAL-vocabulary-20241209/#alternativeText) that neither of those phrases capture on their own?

**Comment by tombaker** [comment]
Discussion continues on proposal of June 3 at https://github.com/dcmi/dc-srap/issues/103
